### PR TITLE
fix(docs-447): clarify the use of shared app values from Score workloads

### PR DESCRIPTION
**Pull request by astromechza**
This is supporting https://github.com/humanitec/developers-docs/pull/740 and https://humanitec.atlassian.net/browse/DOCS-447

### CODE REVIEW

File: score/environment-variables/README.md [hunk 1]
```diff
@@ -2,8 +2,38 @@ Define environment variables for a container. There are two ways to configure th
...
+    spec:
+      containers:
+        main:
+          variables:
+            KEY_THREE: $${values.VALUE}
+            KEY_FOUR: goodbye
+    ```
+
+{{% example-file filename="humanitec.score.yaml" dir="score/environment-variables" githubUrl="https://github.com/humanitec-architecture/example-library/blob/main" %}}
+{{% example-file filename="score.yaml" dir="score/environment-variables" githubUrl="https://github.com/humanitec-architecture/example-library/blob/main" %}}
```
Comment (TobiasBabin) [inline score/environment-variables/README.md:39]: Same as above
```suggestion
```

**Review by TobiasBabin — COMMENTED** [review]
Great explanation and value, thanks so much! I have placed just two changes as per the commit suggestions, a pushed a [change](https://github.com/humanitec-architecture/example-library/pull/98/commits/d8eca2a4aea54666bf5a9c06c6fe0e68d6ec248a) to the Score extensions file which gave me deployment errors in the previous format with the variables in a YAML array. I changed it to the format that's shown in the README as well, which works.

**Comment by astromechza** [comment]
> Great explanation and value, thanks so much! I have placed just two changes as per the commit suggestions, a pushed a [change](https://github.com/humanitec-architecture/example-library/pull/98/commits/d8eca2a4aea54666bf5a9c06c6fe0e68d6ec248a) to the Score extensions file which gave me deployment errors in the previous format with the variables in a YAML array. I changed it to the format that's shown in the README as well, which works.

@TobiasBabin merged the suggestions, thanks :)

**Review by TobiasBabin — APPROVED** [review]
LGTM!

**Pull request merged**